src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.internal.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void checkSetSize() {
+        Assertions.assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @Test
+    void contains() {
+        Assertions.assertThat(numbers.contains(1)).isTrue();
+        Assertions.assertThat(numbers.contains(2)).isTrue();
+        Assertions.assertThat(numbers.contains(3)).isTrue();
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void refactoring_contains(int input) {
+        org.junit.jupiter.api.Assertions.assertTrue(numbers.contains(input));
```

Comment:
```suggestion
        Assertions.assertThat(numbers).contains(input);
```
ValueSource 는 1, 2, 3을 순서대로 매개변수에 넣어 호출하는 방식이라 위와 같은 방식을 사용할 수도 있습니다.